src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Dyslexia fonts
+description: A quick overview of dyslexia fonts
+category: Assistive technology
+author: Hamsa Harcourt
+further_reading:
+  - url: https://accessibility.huit.harvard.edu/design-readability
+    title: Design for readability
+    source: Havard University
+date: 2020-10-28
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - assistivetechnology
+---
+
+According to Oxford Advanced Learner's Dictionary:
+
+<blockquote>
+  <p>
+    Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence.
+  </p>
+</blockquote>
+
+Dyslexia affects about 1 in 5 people worldwide. Its symptoms range in severity from mild to extreme. Since letters, numbers, punctuation, and other characters  are often transposed by Dyslexia, font choice is an important thing to consider.
+
+
+## Background
+
+When designing fonts, some typographers flip and rotate one character to form other characters. Letters b, p, d and q are usually created using this method. This is because it makes it easier to make the font. However, this practice can causes problems for people with Dyslexia. This is because some letters, numbers, punctuation, and other characters become indistinguishable. Hence, making it easy to confuse one for another. As a result, people with Dyslexia prefer fonts where all the characters are unique. 
+
+
+## Introducing Dyslexia fonts
+
+Dyslexia fonts are typefaces designed specially for people with Dyslexia. They are designed so that each character is unique and distinguishable. Dyslexia fonts also make each character unique by using thicker lines in some parts of the letters, as well as slanting them a bit. 
```

Comment:
```suggestion
Dyslexia fonts are typefaces designed so that each character is unique and distinguishable. Dyslexia fonts also make each character unique by using thicker lines in some parts of the letters, as well as slanting them a bit. They are intended to help people with Dyslexia.
```

Updating the copy a bit to emphasize what the desired outcome is for their designers. What the truth is may be different.